CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.10)
 project(vr-update CXX)
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 23)
```

Comment:
Commit message:    SP7: build: upgrade to C++23

Reply:
Ack